roachtest: hibernate-spatial failed

roachtest.hibernate-spatial [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/16082126?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/16082126?buildTab=artifacts#/hibernate-spatial) on master @ [eccb77d2320955b477bb03bc1f1aee137bd1de76](https://github.com/cockroachdb/cockroach/commits/eccb77d2320955b477bb03bc1f1aee137bd1de76):


```
(hibernate.go:173).func1: all attempts failed for building hibernate (without tests): full command output in run_213436.530230839_n1_cd-mntdata1hibernate.log: COMMAND_PROBLEM: exit status 1
test artifacts and logs in: /artifacts/hibernate-spatial/run_1
```

Parameters:
 - <code>ROACHTEST_arch=amd64</code>
 - <code>ROACHTEST_cloud=azure</code>
 - <code>ROACHTEST_coverageBuild=false</code>
 - <code>ROACHTEST_cpu=4</code>
 - <code>ROACHTEST_encrypted=false</code>
 - <code>ROACHTEST_fs=ext4</code>
 - <code>ROACHTEST_localSSD=true</code>
 - <code>ROACHTEST_metamorphicBuild=false</code>
 - <code>ROACHTEST_ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

_Grafana is not yet available for azure clusters_</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*hibernate-spatial.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-40315

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':documentation'.
> Could not resolve all files for configuration ':documentation:classpath'.
   > Could not find com.burgstaller:okhttp-digest:1.10.
     Searched in the following locations:
       - https://plugins.gradle.org/m2/com/burgstaller/okhttp-digest/1.10/okhttp-digest-1.10.pom
       - file:/home/ubuntu/.gradle/tmp/plugins/com/burgstaller/okhttp-digest/1.10/okhttp-digest-1.10.pom
     Required by:
         project :documentation > org.asciidoctor.jvm.gems:org.asciidoctor.jvm.gems.gradle.plugin:3.3.2 > org.asciidoctor:asciidoctor-gradle-jvm-gems:3.3.2 > com.github.jruby-gradle:jruby-gradle-core-plugin:2.0.2 > io.github.http-builder-ng:http-builder-ng-okhttp:1.0.3

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Get more help at https://help.gradle.org.


Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.3/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
BUILD FAILED in 1s
```

--- a/pkg/cmd/roachtest/tests/hibernate.go
+++ b/pkg/cmd/roachtest/tests/hibernate.go
@@ -28,7 +28,7 @@ var hibernateReleaseTagRegex = regexp.MustCompile(`^(?P<major>\d+)\.(?P<minor>\d
 
 // WARNING: DO NOT MODIFY the name of the below constant/variable without approval from the docs team.
 // This is used by docs automation to produce a list of supported versions for ORM's.
-var supportedHibernateTag = "6.3.1"
+var supportedHibernateTag = "6.6.0"
 
 type hibernateOptions struct {
 	testName string

